packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -10,6 +10,8 @@ import moment from 'moment';
 import { isEAForum } from '../../../lib/instanceSettings';
 import { useCookiesWithConsent } from '../../hooks/useCookiesWithConsent';
 import { PODCAST_TOOLTIP_SEEN_COOKIE } from '../../../lib/cookies/cookies';
+import Button from '@material-ui/core/Button';
+import { useDialog } from '../../common/withDialog';
```

Comment:
Pretty sure all the changes in this file are unused

Reply:
Good catch, removed